modules/azure/logic_app_trigger_http_request_data/main.tf

```diff
@@ -0,0 +1,28 @@
+terraform {
+  required_version = ">=1.1.2"
+
+  required_providers {
+    azapi = {
+      source = "azure/azapi"
+      version = "1.0.0"
+    }
+    azurerm = "=3.10.0"
+  }
+
+  backend "azurerm" {}
+}
+
+provider "azapi" {
+}
+
+provider "azurerm" {
+  features {}
+}
+
+// See also: https://github.com/hashicorp/terraform-provider-azurerm/issues/18866
+data "azapi_resource_action" "logicapp_callbackurl" {
```

Comment:
These "=" should be aligned when you have the correct formatter installed and format on save is on. I am not sure what the exact settings are in intellij.

Reply:
thx, reformatted the code in this file, and enabled autoformatting on save.